lib/indexer/compare_enumerator.rb

```diff
@@ -74,7 +74,7 @@ def client
     end
 
     def search_config
-      @search_config ||= SearchConfig.new
+      @search_config ||= SearchConfig.instance
```

Comment:
What does this give `SearchConfig` that is different from `new`?

Reply:
I think I did this to help with testing as it gets the stubs on the instance by default in integration tests. But I am not 100% sure. The `instance` method just returns a memorised copy of `new`